internal/function_time_parser.go

```diff
@@ -416,6 +416,10 @@ func weekOfDayFormatter(t *time.Time) ([]rune, error) {
 	return []rune(dayOfWeeks[int(t.Weekday())]), nil
 }
 
+func weekOfDayNumberFormatter(t *time.Time) ([]rune, error) {
+	return []rune(fmt.Sprint(int(t.Weekday())))
```

Comment:
Thanks for the update @philschoefer 
It looks like the build is failing as you're no longer returning a `nil` error

By the way, I'm not a project member, so I won't have permission to approve / get this merged.
We'll need to wait for @goccy